test/e2e/service_test.go

```diff
@@ -91,6 +93,9 @@ func TestService(t *testing.T) {
 
 	t.Log("create services with volume mounts and subpaths")
 	serviceCreateWithMount(r)
+
+	t.Log("create service with scale activation")
+	serviceCreateActivation(r, "svc1", 2)
```

Comment:
The test might be suitable for `service_options_test.go`also with https://github.com/knative/client/blob/main/test/e2e/service_options_test.go#L340.